docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Cycles Minting Canister (CMC) API
+
+<MarkdownChipRow labels={["Reference"]} />
+
+The Cycles Minting Canister (CMC) is a system canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
```

Comment:
```suggestion
The Cycles Minting Canister (CMC) is a protocol canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
```

(during the last renaming of proposal topics, we were asked to call them "protocol canister"